load-test/creditor_institutions.js

```diff
@@ -0,0 +1,213 @@
+	deleteCreditorInstitution,
+	getEncodings,
+	createEncodings,
+	deleteEncodings,
+	getIbans,
+	getStationsRelationship,
+	createStationRelationship,
+	updateStationRelationship,
+	deleteStationRelationship
+} from "./helpers/creditor_institutions_helper.js";
+
+import {
+	getStationCode,
+	createStation,
+	deleteStation
+} from "./helpers/station_helper.js";
+
+import {
+	getBrokerCode,
+	createBroker,
+	deleteBroker
+} from "./helpers/ci_broker_helper.js";
+
+// read configuration
+const varsArray = new SharedArray('vars', function () {
+	return JSON.parse(open(`./${__ENV.VARS}`)).environment;
+});
+// workaround to use shared array (only array should be used)
+const vars = varsArray[0];
+const rootUrl = `${vars.host}/${vars.basePath}`;
+
+
+export function setup() {
+	// 2. setup code (once)
+	// The setup code runs, setting up the test environment (optional) and generating data
+	// used to reuse code for the same VU
+
+	// precondition is moved to default fn because in this stage
+	// __VU is always 0 and cannot be used to create env properly
+}
+
+function precondition(params, id) {
+	const tempId = `CI${id}`;
+
+	// remove the creditor institution if it already exists
+	let response = deleteCreditorInstitution(rootUrl, params, id);
+	let key = `initial step for creditor institution ${getCiCode(id)}`;
+	check(response, {
+		[key]: (r) => r.status === 200 || r.status === 404,
+	});
+
+	// create a broker in order to create the following station
+	response = createBroker(rootUrl, params, tempId);
+	key = `initial step for station-broker ${getStationCode(id)} / ${getBrokerCode(tempId)}`;
+	check(response, {
+		[key]: (r) => r.status === 201 || r.status === 404,
+	});
+
+	sleep(0.5)
+
+	// create a station in order to create relationship with creditor institution
+	// id is created with CI to avoid collision with stations script
+	response = createStation(rootUrl, params, tempId);
+	key = `initial step for ci-station relationship 1/2 ${getCiCode(id)} / ${getStationCode(tempId)}`;
+	check(response, {
+		[key]: (r) => r.status === 201 || r.status === 404,
+	});
+
+	// remove ci-station relationship
+	response = deleteStationRelationship(rootUrl, params, id, tempId);
+	key = `initial step for ci-station relationship 2/2 ${getCiCode(id)} / ${getStationCode(tempId)}`;
+	check(response, {
+		[key]: (r) => r.status === 200 || r.status === 404,
+	});
+}
+
+function postcondition(params, id) {
+	const tempId = `CI${id}`;
+
+	// remove station and broker used in the test
+	let response = deleteStation(rootUrl, params, tempId);
+	let key = `final step for ci-station relationship ${getCiCode(id)} / ${getStationCode(tempId)}`;
+	check(response, {
+		[key]: (r) => r.status === 200 || r.status === 404,
+	});
+
+	response = deleteBroker(rootUrl, params, tempId);
+	key = `final step for ci-broker ${getCiCode(id)} / ${getBrokerCode(tempId)}`;
+	check(response, {
+		[key]: (r) => r.status === 200 || r.status === 404,
+	});
+}
+
+export default function (data) {
```

Comment:
if as i saw from your case , u don't use
![image](https://user-images.githubusercontent.com/36746022/169001407-4c30a58b-ad71-43e2-ac18-030f71cf47ae.png)
i think you can remove `data` from default function.